### ISSUE TITLE
`Multilink.update_or_create` bad **kwargs handling?

### ISSUE THREAD

**Issue by stdedos** [gabstopper/smc-python#30]
During running

```python
Multilink.update_or_create(
                name=OUTBOUND_NETLINK_NAME.format(vfw_name, 4),
                multilink_members=multilink_members_ipv4,
                comment=comment.format(vfw_name, 4)
        ).add_category([TAG_NAME])
```

I get:
```python
[...]
  File "smc/base/model.py", line 652, in update_or_create
    element.update()
  File "smc/base/model.py", line 448, in update
    result = request.update()
  File "smc/api/common.py", line 87, in update
    return self._make_request(method='PUT')
  File "smc/api/common.py", line 47, in _make_request
    raise err
smc.api.exceptions.UpdateElementFailed: Invalid JSON format: At line 1 and column 648, multilink_members is not recognized as JSON attribute.
```
----
If I do the naive amend of:
```python
@@ smc/base/model.py:651 @@
            if updated and not defer_update:
                element.data.pop('multilink_members', None)  # This
                element.update()
```

Then parsing continues, and elements (seem) to be created as-expected, e.g.:

![image](https://user-images.githubusercontent.com/133706/43211655-ba4aafc8-903a-11e8-9db7-e74ef9e50abe.png)

**Comment by gabstopper** [comment]
Ah good catch. Some classes require an override of update_or_create to filter attributes or attribute values. I will add this for the develop branch push expected tomorrow.

**Comment by gabstopper** [comment]
Resolved in latest develop branch >= 0.6.2.rc.2
